pkgs/applications/science/logic/ott/default.nix

```diff
@@ -16,7 +16,13 @@ stdenv.mkDerivation rec {
 
   installTargets = "ott.install";
 
-  postInstall = "opaline -prefix $out";
+  postInstall = ''
+    opaline -prefix $out
+  ''
+  # There is `emacsPackages.ott-mode` for this now.
+  + ''
+    rm -r $out/share/emacs
+  '';
```

Comment:
```suggestion
    opaline -prefix $out

    # There is `emacsPackages.ott-mode` for this now.
    rm -r $out/share/emacs
  '';
```

Reply:
I prefer to always avoid comments going in the derivations. It causes more rebuilds and wastes space.

Reply:
This is causing a rebuild anyway and the comment does not get any bigger by this. Actually my suggestion saves a line.

Reply:
Sorry I was a bit unclear. I mean it causes rebuilds if the comment alone needs to be changed in the future, and it wastes space in DRV files.